<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QWidget" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="myBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/battlePlayerBoxS.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/battleFoeBoxD.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/battleCommand.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myPokemon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/sprites/pokemon/back/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherPokemon">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/sprites/pokemon/back/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>48</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="myName">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>188</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QLabel" name="myHealthBar">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>212</y>
     <width>81</width>
     <height>7</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/overlay.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherHealthBar">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>72</y>
     <width>81</width>
     <height>7</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/overlay.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>318</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>318</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="boxMessage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/battleMessage.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="boxText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>286</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="SolidPattern">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Pokémon DP Pro</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="myBall1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myBall2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myBall3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myBall4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myBall5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="myBall6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="otherBall5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/ui/img/ballstatus.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="box_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-90</y>
     <width>561</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="qressource.qrc">:/img/bg/Sky.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>box_2</zorder>
  <zorder>myBox</zorder>
  <zorder>otherBox</zorder>
  <zorder>myPokemon</zorder>
  <zorder>otherPokemon</zorder>
  <zorder>otherName</zorder>
  <zorder>myName</zorder>
  <zorder>myHealthBar</zorder>
  <zorder>otherHealthBar</zorder>
  <zorder>myBall1</zorder>
  <zorder>myBall2</zorder>
  <zorder>myBall3</zorder>
  <zorder>myBall4</zorder>
  <zorder>myBall5</zorder>
  <zorder>myBall6</zorder>
  <zorder>otherBall1</zorder>
  <zorder>otherBall3</zorder>
  <zorder>otherBall4</zorder>
  <zorder>otherBall2</zorder>
  <zorder>otherBall6</zorder>
  <zorder>otherBall5</zorder>
  <zorder>box</zorder>
  <zorder>button4</zorder>
  <zorder>button2</zorder>
  <zorder>button1</zorder>
  <zorder>button3</zorder>
  <zorder>boxMessage</zorder>
  <zorder>boxText</zorder>
 </widget>
 <resources>
  <include location="qressource.qrc"/>
 </resources>
 <connections/>
</ui>
